x-pack/plugins/monitoring/server/lib/cluster/get_clusters_from_request.ts

```diff
@@ -168,17 +169,40 @@ export async function getClustersFromRequest(
     }
   }
   // add kibana data
-  const kibanas =
+  const [kibanas, kibanaClusterRules, kibanaInstanceRules] =
     isInCodePath(codePaths, [CODE_PATH_KIBANA]) && !isStandaloneCluster
-      ? await getKibanasForClusters(req, clusters, CCS_REMOTE_PATTERN)
-      : [];
+      ? await Promise.all([
+          getKibanasForClusters(req, clusters, CCS_REMOTE_PATTERN),
+          getClusterRuleDataForClusters(req, clusters, CCS_REMOTE_PATTERN),
+          getInstanceRuleDataForClusters(req, clusters, CCS_REMOTE_PATTERN),
+        ])
+      : [[], []];
```

Comment:
```suggestion
      : [[], [], []];
```
Should all of them default to array when there is no data or does `kibanaInstanceRules` need to be `undefined`?